tests/dataplicity/m2m/wsclient/test_channelfile.py

```diff
@@ -0,0 +1,33 @@
+from dataplicity.m2m.wsclient import ChannelFile
+
+
+class FakeClient(object):
+    """ The ChannelFile class calls
+        `self.client.channel_write` inside, therefore this simple class can
+        act as a client representative.
+    """
+    def __init__(self):
+        self.buffer = ''
```

Comment:
Yes, since on Py2 `b''` is treated as `str('')` and `sys.stdout` is perfectly happy with `str()` ;)